tests/rptest/services/admin.py

```diff
@@ -67,7 +67,7 @@ def __init__(self,
         if auth is not None:
             self._session.auth = auth
 
-        self._default_node = default_node
+        self._default_node: Node = default_node
```

Comment:
what is `Node`?

Reply:
Must have cherry picked the wrong one. I already fixed this--will update.

Reply:
Done. Thank you sir.